src/components/input.js

```diff
@@ -301,6 +305,32 @@ export const TextArea = ({ onChange, autosize = false, nativeOnChange = false, .
   }, props))
 }
 
+/**
+ * Utility function for wrapping text on periods.
+ */
+export const wrappableOnPeriods = _.flow(str => str?.split(/(\.)/), _.flatMap(sub => sub === '.' ? [wbr(), '.'] : sub))
```

Comment:
This was an existing line of code from LaunchAnalysisModal (I take no credit for it…).

Reply:
```suggestion
const PeriodWrapper = ({ children }) => {
  const child = Children.only(children)
  const wrappableText = _.flow(
    _.split(/(\.)/),
    _.flatMap(sub => sub === '.' ? [wbr(), '.'] : sub)
  )(child)
  return h(Fragment, wrappableText)
}
```

Reply:
We ended up removing this because it wasn't actually useful for textual messages (since HTML wraps on whitespace by default).